man/systemd-hibernate-resume.service.xml

```diff
@@ -17,12 +17,14 @@
 
   <refnamediv>
     <refname>systemd-hibernate-resume.service</refname>
+    <refname>systemd-hibernate-resume-clear-efi.service</refname>
     <refname>systemd-hibernate-resume</refname>
     <refpurpose>Resume from hibernation</refpurpose>
   </refnamediv>
 
   <refsynopsisdiv>
     <para><filename>systemd-hibernate-resume.service</filename></para>
+    <para><filename>systemd-hibernate-resume-clear-efi.service</filename></para>
```

Comment:
here too, i'd drop the "-efi" suffix, for same reson as before.

Also, there's no resuming here, hence maybe call this just `systemd-hibernate-clear.service`?